fv3gfs/util/io.py

```diff
@@ -1,5 +1,5 @@
 import cftime
-import xarray as xr
+from . import _xarray
```

Comment:
As a user and reader of the code, I might prefer to see this instead as `from . import _xarray as xr`

Reply:
I've switched the naming in all three modules as suggested.